lib/array_intersection.rb

```diff
@@ -1,6 +1,27 @@
 # Returns a new array to that contains elements in the intersection of the two input arrays
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: n*m or quadratic time complexity that is dependent on length of smaller array
+# Space complexity: Up to On after sorting (in place is possible) where n is length of smaller array
```

Comment:
I believe space complexity is O(1) for quadratic time complexity/insertion sort. 
Source 1: [Comparison Sort](https://en.wikipedia.org/wiki/Sorting_algorithm#Comparison_sorts)
Source 2: [Time Complexity](https://en.wikipedia.org/wiki/Time_complexity#Sub-quadratic_time)